### PR TITLE
Fixed bug on carousel that resulted from not appropriately updating s…

### PR DESCRIPTION
**Pull request by dprevoznik**
…hownIndices variable after a product is deleted. Also added styling to make Cards the same height. Lastly, verticlaly centered "Add +" card and directional buttons.

### CODE REVIEW

File: src/components/productDetails/addCard.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.is-vertical-centered {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.addCardButton {
+    padding: 0;
+    border: none;
```
Comment (RyanTFulmer) [inline src/components/productDetails/addCard.css:9]: nice styling

File: src/components/productDetails/addCard.jsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
     <div class="column is-2">
-      <div class="card">
-        <button value="Add" onClick={e => onClickButton("Add", pageProduct.id)}>
-          <icon>{"+++ADD 2 OUTFIT+++"}</icon>
+      <div class="card is-vertical-centered has-background-grey-lighter">
+        <button
+          class="button is-medium is-vertical-centered addCardButton"
+          value="Add"
+          onClick={e => onClickButton("Add", pageProduct.id)}
+        >
```
Comment (RyanTFulmer) [inline src/components/productDetails/addCard.jsx:12]: good changed styling, but maybe make a better name than "onclickbutton".... maybe onClickAddOutfit?

File: src/components/productDetails/directionalButton.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.is-vertical-centered {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (RyanTFulmer) [inline src/components/productDetails/directionalButton.css:4]: i really need to pick your brain on this, i'm having a lot of issues for vertical and horizontal alignment

File: src/components/productDetails/list.jsx [hunk 4]
```diff
@@ -77,11 +77,17 @@ const List = ({
...
       fetchOutfits(setOutfits);
       fetchOutfitsImages(setOutfitsImages);
+      // on delete - move all shown indices down one if first shown index doesn't equal zero.
+      if (shownIndices[0] !== 0) {
+        let newShownIndices = shownIndices.map(idx => idx - 1);
+        setShownIndices(newShownIndices);
+      }
     }
   }
   return (
```
Comment (RyanTFulmer) [inline src/components/productDetails/list.jsx:93]: good bug fix

**Review by RyanTFulmer — APPROVED** [review]

File: src/components/productDetails/addCard.jsx (hunk 2 above)
Comment (dprevoznik) [reply src/components/productDetails/addCard.jsx:12]: good idea. I will definitely simplify some later confusion that way upon looking over my code again.

**Pull request merged**